.gitignore

```diff
@@ -3,3 +3,6 @@ composer.phar
 composer.lock
 phpunit.xml
 .phpcs-cache
+.idea
+.history
+.gitignore
```

Comment:
Please remove changes to this file: `.idea` and `.history` should be handled in your personal ignore file (see https://help.github.com/en/github/using-git/ignoring-files#create-a-global-gitignore) as adding a bunch of IDE configs doesn't make sense for projects. `.gitignore` just shouldn't be there at all ;)